### PR TITLE
fix(layout): use umami layout

### PR DESCRIPTION
**Pull request by joaogarin**
Just changing from contenta branding to the Umami magazine. Nothing fancy going on here so will merge feel free to take a look still afterwards. just want to merge to proceed to laying out list, cards and detail

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Contenta Angular",
-  "short_name": "Contenta",
+  "name": "Umami Magazine",
```
Comment (dawehner) [inline src/manifest.json]: I personally believe we should mention contenta still somewhere on the website :)

Comment (e0ipso) [inline src/manifest.json]: Yes. They plan to have a "Powered by Contenta" footer item. They mentioned in Slack yesterday.

**Comment by dawehner** [comment]
Cool, I guess we could use that in any frontend then?

Mateu Aguiló Bosch <notifications@github.com> schrieb am Fr., 23. Juni 2017
um 09:55 Uhr:

**Comment by e0ipso** [comment]
> Cool, I guess we could use that in any frontend then?

A good recommendation. Can you talk to @justafish to ask her if she'll add it?

**Comment by dawehner** [comment]
Sure

Mateu Aguiló Bosch <notifications@github.com> schrieb am Fr., 23. Juni 2017
um 10:27 Uhr:

**Comment by joaogarin** [comment]
Right we will def have a "Powered by contenta" somewhere (not sure still where) and we will also use the contenta logo in some about us page I imagine!

So it will definitely be there!

**Pull request merged**